docs/introduction/resources.md

```diff
@@ -22,6 +22,15 @@ is a model. Your library may have two copies that you can check out. Those are r
 
 *Names TBD.
 
+Resource categories are a Taxonomy. That means they can be defined very broadly and generally and maybe vaguely, or they can be defined very narrowly, but fit into broader categories. 
+
+So, for example, you may want an apple. Or you may want a green apple. Or you may want a Granny Smith apple.
+Or you may want a Granny Smith apple from your neighboring farm.
```

Comment:
Is this last one a valid example? I dont think the location is relevant for the resource category.